.d-compiler

```diff
@@ -1 +1 @@
-dmd-2.085.0
```

Comment:
BTW this file was for Heroku AFAIR so we can probably rm it

Reply:
That's OK, I use it for the scripts on the server with the same purpose.